src/lib/transaction_snark_work/transaction_snark_work.ml

```diff
@@ -113,6 +113,10 @@ include T
 
 type unchecked = t
 
+module Unchecked = struct
```

Comment:
This is not required. remove? 
`Transaction_snark_work.t` is the unchecked version

Reply:
I need this since the rpc types need to be versioned.

Reply:
Not sure I understand. Transaction_snark_work.t is versioned, right?

Reply:
oh i see what you are saying! i changed it. I thought you were hard set on using the unchecked type. type t certainly works.